lib/SQLSyntax.lean

```diff
@@ -58,6 +59,7 @@ syntax " OUTER " : join
 
 declare_syntax_cat                                 sqlFrom
 syntax ident                                     : sqlFrom
+syntax sqlFrom ", " sqlFrom                      : sqlFrom
```

Comment:
I didn't know this way of performing joins. Can it take more than two tables like `t₁, t₂, t₃, ...`? If so, then we might want `syntax sqlFrom,+ : sqlFrom` instead